src/Router.svelte

```diff
@@ -100,6 +101,10 @@
   // will not find an active Route in SSR and in the browser it will only
   // pick an active Route after all Routes have been registered.
   $: {
+    if (!caseSensitive) {
+      $location.pathname = $location.pathname.toLowerCase();
+    }
+
     const bestMatch = pick($routes, $location.pathname);
```

Comment:
Don't you also need to make sure all the `$routes` we're comparing against are lowercase?  I think that probably the better solution is for `function pick` to lowercase the values before it compares them, without actually modifying the declared routes or current pathname.  Then... we should also probably expose a helper function to make it easy to compare routes in JS (e.g. to detect "active" links) — something like `isCurrent(href, {location: $location, caseSensitive: false})` with sensible defaults for $location and caseSensitive.